preview.netcdf/ncsa.geo.netcdf.extractor.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+"""Example extractor based on the clowder code."""
+
+import logging
+import subprocess
+import json
+import pyclowder
+from pyclowder.extractors import Extractor
+import pyclowder.files
+import os
+import netCDF4 as nc
+import sys
+from netCDF4 import Dataset as Dataset
+import matplotlib.pyplot as plt
+import numpy as np
+import mpl_toolkits
+import netcdfutils
+from mpl_toolkits.basemap import Basemap
+plt.rcParams['figure.figsize'] = (16.0, 12.0)
+
+
+
```

Comment:
Sorry for nitpicking... there are two many blank lines here. Could you try "black" or IDE reformatting

Reply:
did formatting